database/migrations/2022_03_29_120757_create_newsletter_subscribers_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    /**
+     * Run the migrations.
+     */
```

Comment:
```suggestion
```

Reply:
Remove that comments.

Reply:
@kamilpiech97, maybe you could implement something like this for our Codestyle package? https://github.com/kubawerlos/php-cs-fixer-custom-fixers/blob/main/src/Fixer/NoDoctrineMigrationsGeneratedCommentFixer.php